### ISSUE TITLE
Barely visible cause dots are black and the chest is black

### ISSUE THREAD

**Issue by ghost** [Buuz135/FindMe#22]
This is netherite barrel from iron barrels mod
![image](https://user-images.githubusercontent.com/10179239/126892438-8e792c0d-27ce-49ab-ad8c-ffc030f9b52f.png)

**Comment by ghost** [comment]
```

[COMMON]
	#Checking radius centered on the player
	#Range: > 0
	RADIUS_RANGE = 8
	#Checks item damage or not
	IGNORE_ITEM_DAMAGE = false
```
I suggest to make color for barrels configurable or make it white in the same way as chests

**Comment by Buuz135** [comment]
Fixed in 1.18
